crates/bevy_ecs/src/system/system.rs

```diff
@@ -17,6 +17,11 @@ impl SystemId {
 }
 
 /// An ECS system that can be added to a [Schedule](crate::schedule::Schedule)
+///
+/// Systems are just ordinary Rust functions, each of whose parameters implement the [IntoSystem] trait.
```

Comment:
```suggestion
/// Systems are just ordinary Rust functions, where each parameter implements the [SystemParam] trait.
```
This flows much better, and references the correct Trait.